testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -1238,10 +1238,10 @@
 end
 
 When(/^I enter "([^"]*)" hostname on grafana's host field$/) do |host|
-  step 'I click on "var-hostname"'
+  grafana_url = current_url
```

Comment:
Potential problem if we are not on the correct page beforehand

Reply:
If the previous step didn't fail we are in the right place. We rely on that fact a lot in our testsuite.